src/KCFinderAsset.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @copyright Copyright (c) 2017 Nabi KaramAliZadeh <NabiKAZ@gmail.com>
```

Comment:
Dear @NabiKAZ, do you mind adding the 2amigos info in addition to yours? We understand you should be credited for your work and time + include our gratitude to improve this library, so there is not need to remove yours but would be nice if we keep consistency with other parts of the code.

Reply:
I don't have any problem with this issue, You can change it any way you think it is appropriate.

Reply:
Ok, thanks for your understanding